clippy_utils/src/lib.rs

```diff
@@ -586,14 +586,19 @@ pub fn def_path_res(cx: &LateContext<'_>, path: &[&str], namespace_hint: Option<
         }
     }
 
-    fn find_crate(tcx: TyCtxt<'_>, name: &str) -> Option<DefId> {
-        tcx.crates(())
+    fn find_crate(cx: &LateContext<'_>, name: &str) -> Option<DefId> {
+        cx.tcx
+            .crates(())
             .iter()
             .copied()
-            .find(|&num| tcx.crate_name(num).as_str() == name)
+            .find(|&num| cx.tcx.crate_name(num).as_str() == name)
             .map(CrateNum::as_def_id)
     }
```

Comment:
This change looks like it became no longer necessary

Reply:
Reverted.